.github/workflows/release.yml

```
@@ -0,0 +1,18 @@
+---
+name: sonar-release
+# This workflow is triggered when publishing a new github release
+# yamllint disable-line rule:truthy
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  release:
+    permissions:
+      id-token: write
+      contents: write
+    uses: SonarSource/gh-action_release/.github/workflows/main.yaml@v5
+    with:
+      publishToBinaries: true  # disabled by default
+      mavenCentralSync: true   # disabled by default
```

```suggestion
      publishToBinaries: true  # disabled by default
      mavenCentralSync: true   # disabled by default
      slackChannel: team-sonarqube-build
```
We can add the notification to the channel like the other projects